src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symbol.java

```diff
@@ -435,6 +435,10 @@ public boolean isFinal() {
         return (flags_field & FINAL) != 0;
     }
 
+    public boolean isUnnamed() {
```

Comment:
Should this method get the '@Override @DefinedBy(Api.LANGUAGE_MODEL)' markup?

Reply:
Note that `Element` does not declare `isUnnamed`.

`isUnnamed` overrides method in `VariableElement` (javax.lang.model.element) via subclass `VarSymbol` in `Symbol`.

Reply:
> Note that `Element` does not declare `isUnnamed`.
> 
> `isUnnamed` overrides method in `VariableElement` (javax.lang.model.element) via subclass `VarSymbol` in `Symbol`.

I think it is preferable for VarSymbol.isUnnamed to be deleted; it is functionally the same as the default method defined on VariableElement.

Reply:
> 

PS Just saw the additional discussion on this point below.